src/widget/wtrackmenu.cpp

```diff
@@ -1725,10 +1733,19 @@ void WTrackMenu::slotRemoveFromDisk() {
         delListWidget->addItems(locations);
         mixxx::widgethelper::growListWidget(*delListWidget, *this);
         // Warning text
+        QString delWarningText;
+        if (m_pTrackModel) {
+            delWarningText = tr("Permanently delete these files from disk?") +
+                    QString("<br><br><b>") +
```

Comment:
This and below can become a QStringLiteral

Reply:
done.